.travis.yml

```diff
@@ -8,11 +8,13 @@ os: linux
 
 before_install:
   - |
-    git fetch origin $TRAVIS_BRANCH
-    git diff --name-only FETCH_HEAD | grep -qvE '(\.md$)|(^(doc|documentation|manual|olm|ci|deploy/cr/|deploy/olm-catalog|build/cekit|build/jenkins|build/minikube|test-integration|.gitignore))/' || {
-      echo "Only docs, manual steps, test-integration,CI and OLM files were updated, stopping build process."
-      exit
-    }
+    if [ "$TRAVIS_PULL_REQUEST" != "false" ] ; then
+       git fetch origin $TRAVIS_BRANCH
+       git diff --name-only FETCH_HEAD | grep -qvE '(\.md$)|(^(doc|documentation|manual|olm|ci|deploy/cr/|deploy/olm-catalog|build/cekit|build/jenkins|build/minikube|test-integration|.gitignore))/' || {
+         echo "Only docs, manual steps, test-integration,CI and OLM files were updated, stopping build process."
+         exit
```

Comment:
@rigazilla unfortunately trick with exit inside `before_install` doesn't work correctly in Travis CI anymore. We should find another way for this case
I filled issue #694

Reply:
Yep, that's why I've added the logic to exit only on PR branch.
Anyway I've noticed that the `exit` statement breaks the build only for the master branch.
See build on 2.0.x:
https://travis-ci.org/github/infinispan/infinispan-operator/jobs/753160871
don't know why, I'm looking into

Reply:
@dmvolod  removing these 2 lines:
https://github.com/infinispan/infinispan-operator/blob/aa20bfbdcf360e9733718682b5846120ae76c901/.travis.yml#L21-L22
things go better.
They seems to make the `Build system information` fail somewhere. Do we need to specify the architecture?

Reply:
Currently arm64 machines have much less provisioning time. Which errors occurs on this platform?

Reply:
@dmvolod, I don't know exactly what happens, but the first step `Worker information` shows incomplete info and error (no version and unable to resolve localhost), also docker server info are missing. The overall result is that the `exit` statement hangs instead of exit.

Reply:
@rigazilla oh, we could try to use another Linux distribution search for arm64 hack. Will try to find

Reply:
@rigazilla seems to be hangs with exit fixed in #728